include/h2o/file.h

```diff
@@ -26,4 +26,11 @@
 
 h2o_iovec_t h2o_file_read(const char *fn);
 
+/**
+ * creates a temporary file using the fn_template param.
+ * This is a wrapper to mkstemp(3), but there's no way to access the actual filename.
```

Comment:
```suggestion
 * This is a wrapper to mkstemp(3), but the file is unlinked before returning from the function. Therefore, the name of the file is not provided to the caller.
```

I think it might be a good idea to state that callers do not need to unlink the file, rather than implying that they cannot.

Reply:
Sounds very clear 👍